src/test/resources/com/puppycrawl/tools/checkstyle/api/fullident/InputFullIdentTestArrayType.java

```diff
@@ -0,0 +1,7 @@
+package com.puppycrawl.tools.checkstyle.api.fullident;
+
+public class InputFullIdentTestArrayType {
```

Comment:
please follow comments pattern of InputIllegalTypeSameFileName.java, to define configuration, and mark by comment each line with violation.